aea/cli/registry/push.py

```diff
@@ -57,6 +58,12 @@ def push_item(ctx: Context, item_type: str, item_id: PublicIdParameter) -> None:
 
     items_folder = os.path.join(ctx.cwd, item_type_plural)
     item_path = os.path.join(items_folder, item_id.name)
+
```

Comment:
the annotation of the parameter `item_id` should be `PublicId` and not `PublicIdParameter`

Reply:
fix also the docstring parameters

Reply:
It wasn't me who changed it and not in this PR. But okay, I can do that :)